src/verifai/scenic_server.py

```diff
@@ -62,3 +73,160 @@ def _simulate(self, scene):
 
     def terminate(self):
         pass
+
+class DummySampler(VerifaiSampler):
+
+    def nextSample(self, feedback):
+        return self.last_sample, None
+
+@ray.remote
+class SampleSimulator():
+
+    def __init__(self, scenic_path, worker_num, monitor, options={}, use_carla=False,
+    scenario_params={}):
+        print(scenario_params)
+        scenario_params.update({
+            'port': 2000 + 2*worker_num
+        })
+        self.sampler = ScenicSampler.fromScenario(scenic_path, maxIterations=1, **scenario_params)
+        # reset self.sampler.scenario.externalSampler to dummy sampler
+        # that reads argument
+        self.worker_num = worker_num
+        self.sampler.scenario.externalSampler = DummySampler(self.sampler.scenario.externalParams,
+        self.sampler.scenario.params)
+        self.simulator = self.sampler.scenario.getSimulator()
+        self.monitor = monitor
+        extSampler = self.sampler.scenario.externalSampler
+        if extSampler is None:
+            self.rejectionFeedback = None
+        else:
+            self.rejectionFeedback = extSampler.rejectionFeedback
```

Comment:
Do we need any of this rejection stuff here, since you're using the DummySampler?